### DISCUSSION TITLE
Add Dart template?

### DISCUSSION THREAD

**Discussion by jcbhmr** [devcontainers-community/templates#39] [Ideas]
This is a Discussion to **outreach and ask the original vscode-dev-containers maintainers of the "dart" template if it's OK to move it/copy it/put a "dart" template here**.

<details>

https://github.com/microsoft/vscode-dev-containers/tree/main/containers/dart
![image](https://github.com/devcontainers-community/templates/assets/61068799/9155441d-a146-4370-a451-2bad057dd3f6)

commit history https://github.com/microsoft/vscode-dev-containers/commits/main/containers/dart
![image](https://github.com/devcontainers-community/templates/assets/61068799/436e7b28-acde-4850-8f5c-3dd95c4e545d)

So it seems like the creators and maintainers are the people who became the official https://github.com/devcontainers team?

</details>

Hello @samruddhikhandale @grubint @maksimryndin (three most recent committers) 👋

I'm wondering if we could get a positive 👍 or negative 👎 of your opinion on moving/copying the https://github.com/microsoft/vscode-dev-containers/tree/main/containers/dart template to this @devcontainers-community organization? Right now my goal is to transfer/copy/re-implement the top devcontainer templates shown here: https://github.com/orgs/microsoft/packages?sort_by=downloads_desc&tab=packages&sort_by=downloads_desc&q=vscode-dev-containers

cc @eitsupi <sup>Any other information I should include?</sup>

ref #30 #31 
from https://github.com/devcontainers-community/templates/pull/31#discussion_r1225280485

**Comment by maksimryndin** [comment]
Sorry, I am just an occasional contributor. Better to ask @Chuxel (see https://github.com/Chuxel)

**Comment by samruddhikhandale** [comment]
Hi @jcbhmr 👋 

Thanks for asking the question and showing interest, appreciate such support from the community! ✨ 

The [dart Template](https://github.com/microsoft/vscode-dev-containers/tree/main/containers/dart) has the "Community" category, and that's the reason why I did not move it to [devcontainers/templates](https://github.com/devcontainers/templates) during migration. Looking at the commit history/docs for dart, doesn't seem like we have the name of the community author, but the contributor is "The VS Code Team". 🤔 

Would appreciate @Chuxel / @bamurtaugh thoughts on this
- Do you remember/aware of the author or the interest/source here? (Not sure, why we have  it tagged as community)
- If this was added by us (prev. vscode-dev-containers maintainers) , then should this be moved to [devcontainers/templates](https://github.com/devcontainers/templates)?  
- Else, what do you think about moving it to https://github.com/devcontainers-community organization as the interest shown here

**Reply by bamurtaugh** [reply]
Thanks for looping me in! My guess is it may be tagged as VS Code team as the VS Code team maintained it over time, though it may have been (co)-contributed by the community. Moving it to the https://github.com/devcontainers-community organization makes sense to me.

**Reply by eitsupi** [reply]
Thanks all!

From going back through the history, it seems that this was added by @aeschli, a member of Microsoft (and maybe a member of the VS Code theam).
https://github.com/microsoft/vscode-dev-containers/commit/815095b6479cfa5f5a678158e469f2bc1b0cedf1

Hi @aeschli, could you look at this discussion? Thanks.
